src/docs/testing/debugging.md

```diff
@@ -222,6 +222,50 @@ If you use Travis or Cirrus for testing, increase the number of
 available file handlers that they can open by adding the same line to
 flutter/.travis.yml, or flutter/.cirrus.yml, respectively.
 
+### Widgets marked const that should be equal to each other, aren't
+
+In debug mode, you may find that two `const` widgets that should to all
+appearances be equal (because of Dart's constant deduplication) are not.
+
+For example, this code should print 1:
+
+```dart
+print(<Widget>{ // this is the syntax for a Set<Widget> literal
+  const SizedBox(),
+  const SizedBox(),
+}.length);
+```
+
+...because the two constants should be considered equal (and indeed the analyzer will
```

Comment:
Could you make this a standalone sentence? As simple as:

"The two constants should be considered equal..."

"the analyzer will complain" => "the analyzer complaints"